app/routes/index.js

```diff
@@ -52,8 +52,9 @@ router.post('/register', authLimit, csrfProtection, checkCaptcha, (req, res, nex
     if(!__nullCheck(req.body) || !usernameCheck || !emailCheck) return res.sendStatus(500)
 
     let newUser = new User({
-        username: req.body.username,
-        email: req.body.email
+        username: encodeString(req.body.username),
+        email: encodeString(req.body.email),
+        link: encodeString(Math.random().toString(36).replace(/[^a-z]+/g, '').substr(0, 10))
```

Comment:
Wasn't author's profile supposed to be fullname with dashes?

Reply:
This is a temporary link until fullname is written.

Reply:
Otherwise you'd be unable to get to author profile, or to edit profile, to give yourself a fullname.

Reply:
Call it noFullNameLink, or sth like that, so it will be more obvious please.